app/scripts/angular-apimock.js

```diff
@@ -77,38 +99,31 @@ angular.module('apiMock', [])
 			return keys;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
+		// Modified from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
 		function serializeValue(v) {
-			if (angular.isObject(v)) {
-				return angular.isDate(v) ? v.toISOString() : angular.toJson(v);
+			if (angular.isDate(v)) {
+				return toISOString(v);
 			}
+
```

Comment:
Did you see the diff? You commented on line 104 (right above these comments). That's kinda how it was before :D
But that was useless. My test coverage report from back then said it wasn't being tested: https://coveralls.io/builds/3755120/source?filename=app%2Fscripts%2Fangular-apimock.js#L83 (their page scrolls so that line is the first line in the browser and easily missed, so check the line nr in the link ^^)

So obviously I thought that I forgot some cases, which I kinda did and lead to the date tests being added (see `/spec/services/angular-apimock.js`). But another reason was that `isObject`/`toJson` was already avoided because of this: https://github.com/seriema/angular-apimock/blob/9252f7184fa6dc3d855dda89c8f6950e2c8cd129/app/scripts/angular-apimock.js#L131-L137